src/python/pants/backend/helm/goals/deploy_test.py

```diff
@@ -19,6 +19,7 @@
 from pants.backend.helm.testutil import HELM_CHART_FILE
 from pants.backend.helm.util_rules.tool import HelmBinary
 from pants.core.goals.deploy import DeployProcess
+from pants.core.util_rules import stripped_source_files
```

Comment:
I think you shouldn't need these now?

Reply:
it's used for the `helm_deployment` target rather than the `helm_chart` one. But I indeed could just use plain `SourceFiles`  as deployments are more free-form and that way I will remove the need for the end user to configure source roots in that part as well.